syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -1103,3 +1107,5 @@ def numpy_tensor(self):
                 "on a wrapper. Add NumpyTensor to the chain by hand if you want "
                 "this functionality.",
             )
+
+    reconstruct = get
```

Comment:
how is that useful?

Reply:
this was added temporarily as few of the tests are written using reconstruct(), which fails as it gets called on native.py, in future, it will be good to go with .get() instead of reconstruct()

Reply:
I disagree, you "get" a remote tensor but you "reconstruct" a shared tensor, calling everything get is the easy way, not the correct one, 
can you explain further? give an example of a test case that fails?

Reply:
![Screenshot 2020-08-30 at 12 17 27 AM](https://user-images.githubusercontent.com/28955148/91644057-31188900-ea56-11ea-854d-ec4c36297bcb.png)

if that is not included  this error will prompt up with "reconstruct"

Reply:
fixed in the latest commit!